UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSupplies.json

```diff
@@ -0,0 +1,178 @@
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPatientInfo-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPractitionerInfo-prepopulation"
+    }
+  ],
+  "title": "UrologicalSupplies Order Questionnaire",
+  "status": "draft",
+  "date": "2020-06-22",
+  "publisher": "Da Vinci DTR",
+  "item": [
+    {
+      "linkId": "1",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+          "valueCanonical": "questionnaire/patient-info"
+        }
+      ],
+      "type": "display"
+    },
+    {
+      "linkId": "2",
+      "type": "group",
+      "required": false,
+      "text": "Diagnosis",
+      "item": [
+        {
+          "type": "open-choice",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "drop-down",
+                    "display": "Drop down"
+                  }
+                ],
+                "text": "Drop down"
+              }
+            }
+          ],
+          "required": false,
+          "linkId": "2.1",
+          "text": "Patient Diagnosis",
+          "answerOption": [
+            {
+              "valueCoding": {
+                "code": "Y",
+                "display": "Permanent urinary incontinence, not expected to be medically or surgically corrected within 3 mo."
+              }
+            },
+            {
+              "valueCoding": {
+                "code": "Yes",
+                "display": "Permanent urinary retention, not expected to be medically or surgically corrected within 3 months"
+              }
+            },
+            {
+              "valueCoding": {
+                "code": "other",
+                "display": "Other"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "3",
+      "type": "group",
+      "required": false,
+      "text": "Order Start Date",
+      "item": [
+        {
+          "type": "date",
+          "required": false,
+          "linkId": "3.1",
+          "text": "Order start date, if different from date of order"
+        }
+      ]
+    },
+    {
+      "linkId": "4",
+      "required": false,
+      "text": "Order Type",
+      "type": "open-choice",
+      "item": [
+        {
+          "linkId": "4.1",
+          "text": "Type of Order",
+          "type": "open-choice",
+          "answerValueSet": "http://hl7.org/fhir/ValueSet/request-intent"
+        }
+      ]
+    },
+    {
+      "linkId": "5",
+      "type": "group",
+      "required": false,
+      "text": "Order",
```

Comment:
For this group of items, are we supposed to pre-populate the order from the device request? 
For other orders, beside the items from the drop down, need to specify other attributes. You can add a repeatable gtable. Check the comment in your another PR for Supply example. 

![Screen Shot 2020-08-05 at 1 21 19 PM](https://user-images.githubusercontent.com/58572522/89443814-d1f97a00-d71e-11ea-9e57-c78b6b50c8c9.png)